.github/workflows/linux-ci.yml

```diff
@@ -35,22 +35,17 @@ jobs:
         CC: /usr/bin/clang
         CXX: /usr/bin/clang++
     - name: CMake (coverage/clang-tidy/clang-asan)
-      if: github.ref == 'refs/heads/master'
       run: |
-        cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Debug -DTW_CODE_COVERAGE=ON -DTW_ENABLE_CLANG_TIDY=ON -DTW_CLANG_ASAN=ON
-      env:
-        CC: /usr/bin/clang
-        CXX: /usr/bin/clang++
-    - name: CMake (coverage)
-      if: github.ref != 'refs/heads/master'
-      run: |
-        cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Debug -DTW_CODE_COVERAGE=ON
+        cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Debug -DTW_UNITY_BUILD=ON -DTW_CODE_COVERAGE=ON -DTW_ENABLE_CLANG_TIDY=ON -DTW_CLANG_ASAN=ON -GNinja
```

Comment:
Is it safe to enable unity build by default?

Reply:
I advise you to let it off by default in the CMake option. Let it on for CI builds. If you want to build faster locally, you can also enable it.